doctr/travis.py

```diff
@@ -226,20 +227,37 @@ def create_gh_pages():
         return True
     return False
 
+def find_sphinx_build_dir():
+    """
+    Find name of docs folder and build subfolder within sphinx docs directory.
+
+    Locate the sphinx docs directory by looking for a ``conf.py`` file.
+    Then look for the ``build`` folder, by convention either ``build`` or ``_build``
+
+    This is called by :func:`commit_docs` if keyword arg ``built_docs`` is not
+    specified on the command line.
+    """
+    docs_folder = glob.glob('**/conf.py')[0].split('/', 1)[0]
```

Comment:
Are you sure about the `recursive` kwarg?  It works locally for me without it.  I guess I could just make it a `*/conf.py` since that file should only be one level deep